galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
No worries, I was talking about fixes for the bugs you've found while adding more galaxy tests.